html/cross-origin-embedder-policy/resources/sw.js

```diff
@@ -0,0 +1,12 @@
+self.addEventListener('activate', event => {
```

Comment:
s/event/e/ to make it aligned with the fetch event handler.

Reply:
Fixed.